pkg/tools/multiexecutor/utils.go

```diff
@@ -59,8 +63,9 @@ func (e *multiExecutor) AsyncExec(id string, f func()) (ch <-chan struct{}) {
 	return ch
 }
 
-func (e *multiExecutor) Executor(id string) Executor {
-	return executorFunc(func(f func()) <-chan struct{} {
+// Executor - returns Executor by ID
+func (e *MultiExecutor) Executor(id string) ExecutorFunc {
```

Comment:
```suggestion
func (e *MultiExecutor) Executor(id string) Executor {
```

Reply:
`serialize` already depends on `multiexecutor`. We cannot use direct `multiexecutor` -> `serialize` dependency. So the only other option is to add something like `serializectx`, but it really doesn't make sense if we can simply make `ExecutorFunc` public and return it.

Reply:
There should be ExecutorFunc

Reply:
Executor interface moved to other package

Reply:
> Executor interface moved to other package

Dont see this. Could you point a file?

Reply:
> serialize already depends on multiexecutor. We cannot use direct multiexecutor -> serialize dependency. So the only other option is to add something like serializectx, but it really doesn't make sense if we can simply make ExecutorFunc public and return it.

I did not quite follow this

Reply:
I would suggest to just move all common staff with context and itnerfaces from serialize to tools/serialize.

Reply:
> serialize already depends on multiexecutor.

I also don't follow this... I would further suggest we move to using https://github.com/edwarnicke/serialize rather than further clogging up tools (we need less in tools/ not more).